src/components/flat-table/flat-table-expandable.stories.mdx

```diff
@@ -1040,6 +1041,73 @@ truncated styling applied, this is because of the caret icon that is rendered wi
   </Story>
 </Preview>
 
+### Controlled
+
+<Preview>
+  <Story name="controlled">
```

Comment:
We need to ignore this story in `chromatic` and add `cypress` test for that purpose because the only one way to check if it works properly